cl/corpus_importer/management/commands/anon_2020_import.py

```diff
@@ -135,7 +135,10 @@ def merge_or_add_opinions(
         )
 
     # Merge with scrape or add opinion to cluster with harvard
-    if OpinionCluster.objects.get(pk=cluster_id).source == "C":
+    if (
+        OpinionCluster.objects.get(pk=cluster_id).source
+        == SOURCES.COURT_WEBSITE
+    ):
```

Comment:
Better to clean this up (black is not being our friend):

```suggestion
    cluster_source = OpinionCluster.objects.get(pk=cluster_id).source
    if cluster_source == SOURCES.COURT_WEBSITE:
```